### ISSUE TITLE
XPath path for api platform responses failes

### ISSUE THREAD

**Issue by arthurGrinjo** [getinsomnia/insomnia#4258]
### Expected Behavior

Response - reference values from other request's responses.

When trying to reach to the first @id it should respond e.g.: /api/domains/8

XPath: $['hydra:member'][0]['@id']

E.g.
`{
  "@context": "\/api\/contexts\/Domain",
  "@id": "\/api\/domains",
  "@type": "hydra:Collection",
  "hydra:member": [
    {
      "@id": "\/api\/domains\/8",
      "@type": "Domain",
      "name": "Domain1"
    }
  ],
  "hydra:totalItems": 1
}`

### Actual Behavior

Returns an error: Invalid JSONPath query: $['hydra:member'][0]['@id']

### Is there an existing issue for this?

- [X] I have searched the [issue tracker](https://www.github.com/Kong/insomnia/issues) for this problem.

### Insomnia Version

2021.6.0

### What operating system are you using?

Windows

### Operating System Version

Windows 10 Pro 20H2

### Installation method

Download from insomnia.rest

### Last Known Working Insomnia version

2021.5.3

**Comment by dimitropoulos** [comment]
hi @arthurGrinjo 

TLDR: you have to (now) escape the `@` with `` ` ``: e.g.

```
$['hydra:member'][0]['`@id']
```

see: https://github.com/Kong/insomnia/issues/4213 for more discussion as to why.

As has been customary, I'd also like to ask whether `jq` syntax would be a satisfactory alternative to you, in this case, were it available.

**Comment by arthurGrinjo** [comment]
Thanks @dimitropoulos! Works great! Couldn't find the post you mentioned. Sorry for that.
